tools/amd_build/build_amd.py

```diff
@@ -141,10 +141,17 @@
                     f.flush()
                     os.fsync(f)
 
+# Check if the compiler is hip-clang.
+def is_hip_clang():
+    try:
+        return 'HIP_COMPILER=clang' in open('/opt/rocm/hip/lib/.hipInfo').read()
+    except:
+        return False
+
 hipify_python.hipify(
     project_directory=proj_dir,
     output_directory=out_dir,
     includes=includes,
     ignores=ignores,
     out_of_place_only=args.out_of_place_only,
-    hip_clang_launch=args.hip_clang_launch)
+    hip_clang_launch=args.hip_clang_launch or is_hip_clang())
```

Comment:
Is there a need to allow for an override still or should it just be `hip_clang_launch=is_hip_clang()`?

Reply:
no. except to keep some existing script happy. would you like me to clean up the option?

Reply:
if you could clean it up, that'd be nice. assuming of course that you can guarantee that the `is_hip_launch()` function will be reliably detecting `hip-clang` moving forward.

Reply:
I updated the PR to remove the old option. I am pretty sure this is the stable way to identify hip-clang.